### PR TITLE
feat: add subgraph access logs

### PR DESCRIPTION
**Pull request by df-wg**
## Checklist

- [ ] I have discussed my proposed changes in an issue and have received approval to proceed.
- [ ] I have followed the coding standards of the project.
- [ ] Tests or benchmarks have been added or updated.
- [ ] Documentation has been updated on [https://github.com/wundergraph/cosmo-docs](https://github.com/wundergraph/cosmo-docs).
- [ ] I have read the [Contributors Guide](https://github.com/wundergraph/cosmo/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: router/internal/requestlogger/requestlogger.go [hunk 1]
```diff
@@ -199,9 +161,58 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
...
-	if h.fieldsHandler != nil {
-		resFields = append(resFields, h.fieldsHandler(nil, r)...)
+	if h.accessLogger.fieldsHandler != nil {
+		resFields = append(resFields, h.accessLogger.fieldsHandler(h.accessLogger.attributes, nil, r, nil)...)
 	}
 
 	h.logger.Info(path, append(fields, resFields...)...)
 }
+
+func (al accessLogger) getRequestFields(r *http.Request) []zapcore.Field {
```
Comment (StarpTech) [inline router/internal/requestlogger/requestlogger.go:171]: Use a pointer receiver.

File: router/internal/requestlogger/subgraphlogger.go
Comment (StarpTech) [inline router/internal/requestlogger/subgraphlogger.go:1]: Please ensure that we have covered the most important paths in unit tests.

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: router/pkg/config/config.schema.json [hunk 2]
```diff
@@ -547,53 +547,56 @@
...
+                  "$ref": "#/definitions/context_fields"
+                },
+                {
+                  "items": {
+                    "properties": {
+                      "value_from": {
+                        "properties": {
+                          "context_field": {
+                            "enum": [
+                              "operation_name",
```
[the remark this answers is not in the document]
Comment (df-wg) [reply router/pkg/config/config.schema.json:582]: From what I could see, the only one that's not available in the subgraph context is `response_error_message` (it is only set by the completion of the full query, and not the subgraph). I removed it, and we'll look into the error logging as part of the next task

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**